device-types/Cisco/MDS-C9148S.yaml

```diff
@@ -0,0 +1,120 @@
+---
+manufacturer: Cisco
+model: MDS 9148S
+slug: ds-c9148s
+part_number: DS-C9148S
+u_height: 1
+is_full_depth: true
```

Comment:
Please add a comment in markdown format with a link to the manufacturers datasheet, if you are able to find a publicly available one